docs/tutorials/time-series.md

```diff
@@ -0,0 +1,738 @@
+   `TIMESTAMP_TRUNC(purchase.ts) >= TIMESTAMP_TRUNC(Wp)`.
+
+3. **Discard old records**. Feldera garbage collects old records that fall below retention
+   bounds in the background.  Since these records are guaranteed to have no effect on
+   new outputs, it is safe to keep them around until a convenient time when they can be
+   removed with low overhead.  In practice, we incorporate garbage collection in the
+   [log-structured merge tree](https://www.feldera.com/blog/feldera-storage)(LSM) maintenance process.
+
+**Not all SQL operators allow discarding old inputs**. Joins, aggregates, and `DISTINCT`'s
+for example may, depending on their exact join condition or the `GROUP BY` clause,
+require storing their entire inputs permanently.  We list operators for
+which Feldera implements garbage collection [below](#sql-for-time-series-analysis).
+
+**Discarding old records is not equivalent to deleting them**. These are
+fundamentally different operations.  Deleting a record means that any outputs
+derived from it should be updated.  For example, deleting a year-old purchase
+requires recomputing the `daily_max` query for the corresponding date.
+In contrast, a discarded record is conceptually still part of the relation;
+however it will not participate in computing any future incremental updates
+and therefore does not need to be stored.
+
+:::warning
+
+Feldera does not automatically garbage collect [materialized tables and views](/sql/materialized).
+When a table or view is declared as materialized, it indicates an explicit user
+request to store its entire contents, making it accessible for
+[ad hoc queries](/sql/materialized/#ad-hoc-queries).
+However, if the materialized table contains an unbounded time series, it will
+continue to consume storage without limit, even if the associated views only
+require storing a bounded portion of the time series.
+
+:::
+
+
+## Emitting final values of a view with `emit_final`
+
+:::warning
+
+The `emit_final` feature is still experimental, and it may be removed
+or substantially modified in the future.
+
+:::
+
+By default, Feldera updates SQL views incrementally whenever new
+inputs arrive. Such incremental updates include deleting existing
+rows and inserting new rows into the view.  By monitoring these
+updates, the user can observe the most up-to-date results of the
+computation.  However, some applications only need to observe its
+final outputs, i.e., rows that are guaranteed to
+never get deleted or updated.
+
+Consider the following query that computes the sum of all purchases
+for each day.
+
+```sql
+CREATE VIEW daily_total AS
+SELECT
+    TIMESTAMP_TRUNC(ts, DAY) as d,
+    SUM(amount) AS total
+FROM
+    purchase
+GROUP BY
+    TIMESTAMP_TRUNC(ts, DAY);
+```
+
+Suppose that a new `purchase` record arrives every second.  In response,
+Feldera updates the `daily_total` value for the current date,
+deleting the old value and inserting a new one.  The user may observe
```

Comment:
you can point to the blog post about incremental computation which has an example